net-misc/trurl/files/trurl-0.7-fix-makefile.patch

```diff
@@ -1,21 +1,22 @@
-See https://github.com/curl/trurl/pull/158#issuecomment-1512397801.
+Make make respect CFLAGS, fix LDFLAGS order, don't force -g and -Werror
```

Comment:
It's generally preferred to just leave old things in place unless fixing them up intentionally (in which case we should revbump them if it's going to make any difference at runtime, which it won't here). i.e. in this case, I don't see a need to bother with 0.7 given it'll be shadowed by 0.14 and it's so old anyway.

But I don't mind you fixing it as it's clear cut here. Just a note in general.

If going to keep the 0.7 fix, please do that in a separate commit (first?), same PR is fine.

Reply:
> It's generally preferred to just leave old things in place unless fixing them up intentionally (in which case we should revbump them if it's going to make any difference at runtime, which it won't here). i.e. in this case, I don't see a need to bother with 0.7 given it'll be shadowed by 0.14 and it's so old anyway.

This patch is different from the 0.14 one.

> But I don't mind you fixing it as it's clear cut here. Just a note in general.
> 
> If going to keep the 0.7 fix, please do that in a separate commit (first?), same PR is fine.

I can do that.
By the way, if you plan on dropping the 0.7 version anyway because it is too old, I can just drop this new 0.7 patch.

Reply:
Let's just drop 0.7, actually. We don't want anyone using it unless you want to keep it for a bit.